linux/keyman-config/keyman_config/__init__.py

```diff
@@ -7,8 +7,10 @@
     # Alpha versions will work against the staging server so that they
     # can access new APIs etc that will only be available there. The staging
     # servers have resource constraints but should be okay for limited use.
-    KeymanComUrl = 'https://staging-keyman-com.azurewebsites.net'
-    KeymanApiUrl = 'https://staging-api-keyman-com.azurewebsites.net'
+    KeymanComUrl = 'https://keyman-staging.com'
+    KeymanApiUrl = 'https://api.keyman-staging.com'
+    KeymanDownloadsUrl = 'https://downloads.keyman-staging.com'
```

Comment:
Currently we don't have a downloads staging site.

Reply:
We have keyman-staging.com, api.keyman-staging.com and help.keyman-staging.com.

Reply:
ah, ok. Done.